### ISSUE TITLE
WIKI: not able to get the value to switch from default text to Custom Text

### ISSUE THREAD

**Issue by VirtualGaming1** [Lightman314/LightmansCurrency#261]
NOTE: I am using the latest curseforge version for Forge MC 1.20.1
I have read in the wiki with the coin master and I am still not able to get it switch.

what I am trying to do is switching it from "1d6e9g3i3c" to "$169.33"
I am confuse of what I am doing wrong and would like to see for examples sets to show how it should look like on the wiki and 

also there is no command for /lcreload

This is what I have set right now

> "Chains": [
>     {
>       "chain": "emeralds",
>       "name": {
>         "translate": "lightmanscurrency.money.chain.emeralds"
>       },
>       "displayType": "lightmanscurrency:number",
>       "displayFormat": {
> 		  "translate": "${value}"},
>       "displayFormatWordy": {
>       },
>       "InputType": "TEXT",
>       "CoreChain": [
>         {
>           "Coin": "minecraft:gold",
>           "baseUnit": true
>         },
>         {
>           "Coin": "minecraft:emerald_block",
>           "exchangeRate": 9
>         }
>       ]
>     },

**Comment by Lightman314** [comment]
If you read your logs when attempting to load this config, you'd probably see some mention of the fact that the "displayFormatWordy" has no "text" or "translation" field, which is causing it to ignore the entire coin chain due to said parsing error. If you do not wish to include a wordy variant, simply delete the field in its entirety. Also "${value}" from the normal "displayFormat" is not a translation key, so it should be in there under a "text" label, not a "translate" label.


And **MOST IMPORTANTLY**, Github is NOT an appropriate place to ask questions. Please use the discord group linked in my mods description, or the comments section of said mod page to ask such questions in the future.

Github is only to be used for reporting issues or making a formal suggestion, not a place to ask for clarification on how something works.

**Comment by VirtualGaming1** [comment]
> Github is only to be used for reporting issues or making a formal suggestion, not a place to ask for clarification on how something works.

Technically this is an issue and suggestion on the wiki page of the Github as it dose need to be improved as it did not stated about "${value}" have to be under text label and I also stated that there is no command of /lcreload since you do have a link in Readme and I also suggested add in example(s) in the wiki for people read them the best of what they understand and still put it in wrong because it is missing important wording like the "${value}" have to be in Text label

**Comment by Lightman314** [comment]
My apologies, I assumed that how the text inputs have to be formatted could be easily researched independently as they use **vanilla** [json text formatting](https://minecraft.fandom.com/wiki/Raw_JSON_text_format) which has more in-depth documentation on it than I could ever provide, but just to be 100% clear I've updated the relevant entries to be more specific (including a hyperlink to the official wiki page as I did above). I also corrected the note about the reload command, as when I last updated this particular page I had not yet reworked my mods other configs which resulted in updating the command used to reload them.

I should also mention that there *is* an[ example config](https://github.com/Lightman314/LightmansCurrency/tree/main/exampleconfig) for the Master Coin List that does exactly what you're trying to do, except with the default coins (but overall the setup is the same in terms of the display formatting, etc.). I should also note that a link to this is easily findable on my discord server, so if you had gone there someone could/would have been able to redirect you to this example without me even needing to get involved in the conversation (not that I mind answering valid questions, but I have very little free time due to work so if someone else is willing to do it for me I sure as heck won't complain).

As a side note, there's not really too much of a *need* for an example config in the latest versions as with the addition of the emerald coin chain, there is now an existing example in the default config values for how a number display should be formatted assuming you have basic knowledge of how vanilla json text works.

Having said that, I stand by what I said earlier that this is still not the proper place for questions (and I've mentioned this various times over various other closed "issues" on this page). Also, you claim that this is *technically* an issue with the documentation, and while that is mildly correct (the only **actual** issue being the incorrect note about the `/lcreload` command), your initial post was not even close to being worded as such so I think I can officially disqualify your initial post as an "Issue Report" on my incorrect documentation, doubly so since you did not use **either** of the available issue report/feature request templates and instead just deleted the template and typed in random question with no context or organization.

### TLDR
* Small tweaks have been made to the wiki page to make it more clear. Thank you for pointing them out, even if you did so in an incorrect manner.
* An example config *does* exist, you just didn't look for it in the right places. Also there's no
* This is **still** not the place for questions. And no, your initial post doesn't even come close to counting as an "Issue Report" as you completely disregarded the issue report template.
